tests/foreman/ui/test_host.py

```diff
@@ -1970,3 +1970,31 @@ def test_positive_gce_cloudinit_provision_end_to_end(
             gce_client.disconnect()
             skip_yum_update_during_provisioning(
                 template='Kickstart default user data', reverse=True)
+
+
+@tier2
+def test_positive_cockpit(session):
+    """Test whether webconsole button and cockpit integration works
+
+    :id: 5a9be063-cdc4-43ce-91b9-7608fbebf8bb
+
+    :expectedresults: Cockpit page is loaded and displays sat host info
+
+    :CaseLevel: System
+
+    """
+    with session:
+        session.organization.select(org_name='Default Organization')
+        session.location.select(loc_name='Any Location')
+        session.host.open_webconsole(entity_name=settings.server.hostname)
+        wd = session.browser.selenium
+        assert len(wd.find_elements_by_xpath('//iframe')) > 0, 'Unable to find iframe on the ' \
+                                                               'page - are we on the Cockpit page?'
```

Comment:
Please use parentheses instead of backslash:
```python
assert len(wd.find_elements_by_xpath('//iframe')) > 0, (
    'Unable to find iframe on the page - are we on the Cockpit page?'
)
```

Reply:
why?

Reply:
whatever @mirekdlugosz  suggested looks nicer!

Reply:
@rplevka 
1. PEP8 discourages backslashes at the end of line
2. With parentheses it's more readable, as you can have message in one line
3. `black`, which we might start using at one point, will reformat it from backslashes to parentheses anyway

Reply:
fair enough